src/Symfony/Component/AssetMapper/CompiledAssetMapperConfigReader.php

```diff
@@ -11,15 +11,19 @@
 
 namespace Symfony\Component\AssetMapper;
 
+use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\Filesystem\Path;
 
 /**
  * Reads and writes compiled configuration files for asset mapper.
  */
 class CompiledAssetMapperConfigReader
 {
+    private readonly Filesystem $filesystem;
+
     public function __construct(private readonly string $directory)
```

Comment:
Could you please linebreak while on it? Thanks

Reply:
Sure.